src/components/header/MenuBar/MenuBar.vue

```diff
@@ -134,4 +148,12 @@ function showTransferZilDialog() {
     component: TransferZilDialog,
   });
 }
+
+function showAboutDialog() {
+  q.dialog({
+    title: 'Neo Savant 0.1.0',
```

Comment:
Can I (sorry again!) opt for "Neo Savant 2.1.0" ? 
```suggestion
    title: 'Neo Savant 2.1.0',
```